### DISCUSSION TITLE
Oral low dose administration does not run out

### DISCUSSION THREAD

**Discussion by liuxinyang1026** [Open-Systems-Pharmacology/Forum#1461] [Questions & Problems]
![微信图片_20230227103032](https://user-images.githubusercontent.com/43397060/221459147-2dcf003a-2e63-4932-83b7-a8131d27d012.png)
When I use intravenous parameters to run the model for oral administration, the window pops up for 10mg as shown in the picture, but it can be run for 20mg or more, what is the reason for this and how to solve it?

**Comment by tobiasK2001** [comment]
Hi,
 Maybe you have some similar issues like discussed here:

https://github.com/Open-Systems-Pharmacology/Forum/discussions/1449

Best, Tobias

**Comment by Yuri05** [comment]
@liuxinyang1026 Can you attach your project?
